### PR TITLE
Locking phones to portrait during FTUE onboarding

### PR DESCRIPTION
**Pull request by ouchadam**
Part of #4584 

- Makes use of a resource bucket flag to determine if the device is big enough to be considered a tablet and in turn, enable a landscape experience in the FTUE auth onboarding flow

| BEFORE PHONE | AFTER PHONE | AFTER LOGIN | AFTER TABLET |
| --- | --- | --- | --- |
|before-rotation|after-rotation|after-login|tablet-rotation

### CODE REVIEW

File: library/ui-styles/src/main/res/values/tablet.xml [hunk 1]
```diff
@@ -2,5 +2,6 @@
 <resources>
 
     <dimen name="width_percent">1</dimen>
+    <bool name="is_tablet">false</bool>
```
Comment (bmarty) [inline library/ui-styles/src/main/res/values/tablet.xml:5]: weird to see that in a file (that I) named `tablet.xml` :)

**Review by bmarty — APPROVED** [review]
LGTM 👍

**Comment by ouchadam** [comment]
> Can I ask you to add the same rotation lock to the Activity `AnalyticsOptInActivity` please? I do not see any other screen where it could be applied at the moment.
> 
> Thanks!

will do :+1:

**Review by bmarty — APPROVED** [review]
Thanks!

**Pull request merged**